TheMET/TheMET/ViewControllers/ArtistArtsViewController.swift

```diff
@@ -154,8 +152,7 @@ class ArtistArtsViewController: UIViewController, UITableViewDelegate, UITableVi
     
     private func loadObjects(artistName: String) {
         guard let artistName = self.artistName else {
-            self.isSearchFailed = false
-            self.contentStatus = .failed
+            self.contentStatus = .failed(ArtistsArtsLoadingError.artistNameNotFound)
```

Comment:
Actually the artistName is always provided as an argument. Why do you use this guard here?